src/Nokia_LCD.h

```diff
@@ -14,6 +14,12 @@
 #include <stdint.h>
 #include "LCD_Fonts.h"
 
+namespace {
+    // Display constants
+    const uint8_t kDisplay_max_width = 84;
```

Comment:
@platisd the namespace is here

Reply:
Right, missed that. You generally shouldn't put anonymous namespaces in header files.
They can sometimes lead to surprising results! 😅
Use a named namespace instead. 👍